### ISSUE TITLE
Ts build fail TS1110: Type expected.

### ISSUE THREAD

**Issue by 98DE9E1F** [FormidableLabs/victory#2409]
**Describe the bug**

/Users/bafan/tuya/doberman-front/node_modules/victory-core/lib/victory-util/events.d.ts(9,42)
      TS1110: Type expected.

the code  is 
```
export declare type ComponentEventName = `on${Capitalize<string>}`;

````

**Victory version**
Which npm version are you using?
6.14.7
typescript 3.8.3

**Comment by scottrippey** [comment]
TypeScript 4.1 introduced these [Template Literal Types](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-1.html) ... so this issue appears to be due to the older TypeScript version being used.

That being said, I'll investigate how often we use this syntax, and see if we could drop it, to increase TS compatibility.

**Comment by scottrippey** [comment]
@gksander We currently use "Template Literal Types" syntax in 2 places: [ComponentEventName] and [SafeAttribute].  Neither is too critical, and only provides a tiny-bit of type safety, so what do you think about replacing these types with `string`, in order to improve TS compatibility with <4.1?

**Comment by gksander** [comment]
That sounds reasonable to me. I wonder if there's any way to fork a type based on TS version it's being used on 🤔 I'm not aware of any mechanism, but it certainly would be nice to use template literals and then TS just fallback to strings on TS <4.1.

**Comment by scottrippey** [comment]
Fixed in `36.6.1` 👍
